src/main/resources/assets/enigmaticlegacy/lang/zh_cn.json

```
@@ -1006,64 +1006,64 @@
   "gui.enigmaticlegacy.blazing_core_bar_offsetY": "0",
   "gui.enigmaticlegacy.blazing_core__COMMENT": "Two strings above allow to move heat bar title by defining offsetX and offsetY to any positive or negative integers. This might be required for certain localizations more than for others depending on amount of symbols title takes takes to be spelled. You can, alternatively, leave title empty if you can't think of a solution that would look good, so that it just won't be rendered. Four symbols or less fit perfectly without any corrections. The string you read currently is just a commentary for localizators - you need not to translate it.",
   
-  "quote.no_peril_1": "Where there is no peril in the task - there can be no glory in its accomplishment.",
-  "quote.end_doorstep_1": "The doorstep of The End... single step now separates you from it.",
-  "quote.only_because_1": "Only because you have fallen does not yet mean you backed down.",
-  "quote.demise_is_1": "Demise is but a temporary inconvenience for a demigod such as you.",
-  "quote.we_fall_1": "We fall, so that we may learn how to pick ourselves back up.",
-  "quote.you_will_endure_1": "You will endure this loss, and learn from it.",
-  "quote.oblivion_rejects_1": "Oblivion rejects you... make use of such opportunity, while you still can.",
-  "quote.setback_1": "A setback, but not a defeat.",
-  "quote.death_may_1": "Death may impede your conquest, but only broken will can halt it.",
-  "quote.eternity_to_keep_1": "You have eternity to keep trying. Do what it takes.",
-  "quote.violence_calls_1": "Violence calls for vengeance. Return, and dispense it!",
-  "quote.immortal_1": "You are immortal! Return and give that pitiful creature a demonstration of this fact.",
-  "quote.appaling_presence_1": "The shell is gone... but appaling presence will forever remain imprinted upon this realm.",
-  "quote.its_destruction_1": "Its destruction is a small consolation... given the implications of its terrible existence.",
+  "quote.no_peril_1": "倘若过程中没有曲折艰辛，那也就无法成就最终的荣耀。",
+  "quote.end_doorstep_1": "终末之地的门户已经打开…只差迈出最后一步了。",
+  "quote.only_because_1": "跌倒，并不意味着你将就此踌躇不前。",
+  "quote.demise_is_1": "对于如你一样的半神而言，死亡不过是暂时的不便罢了。",
+  "quote.we_fall_1": "我们之所以跌倒，是为了学习如何重新站起来。",
+  "quote.you_will_endure_1": "你将承担这次失败的代价，并从中吸取教训。",
+  "quote.oblivion_rejects_1": "深渊拒绝了你…在你还活着的时候，好好利用这次机会吧。",

```

~~此处触发条件是什么？这里的“深渊拒绝了你”，是指的玩家复活？没有陷入永久的死亡？~~
首先不是深渊，而是湮灭，湮灭基石中的湮灭
看了一下源码，确实是死亡触发。那么这里的意思就是，术石劝解玩家，既然湮灭拒绝了你，那么你要好好利用此次机会，不要浪费了。
```suggestion
  "quote.oblivion_rejects_1": "湮灭拒绝了你……毕竟你是不死的，在你尚未失去不死之身前，好好利用这一归来的机会吧。",

```
while you still can “表示应该在某个时候或某个机会还存在的时候抓住它，因为它可能不会持续太久”，此处需要将暗示的内容具体化出来。
这里涉及到一点背景，非欧立方知道玩家是不死的、不朽的半神，所以这里才会这样说。